src/App.js

```diff
@@ -24,25 +17,10 @@ const App = () => {
 
   const contactRoute = "/contact";
   const teamRoute = "/";
-  const navItems = [
-    <Link to={teamRoute}>{t("Team")}</Link>,
-    <Link to={contactRoute}>{t("Contact")}</Link>,
-  ];
-  const footerNavItems = [
-    <Link to={teamRoute} className="usa-footer__primary-link">
-      {t("Team")}
-    </Link>,
-    <Link to={contactRoute} className="usa-footer__primary-link">
-      {t("Contact")}
-    </Link>,
-  ];
 
   return (
     <Router>
-      <Header extended={true}>
-        <Title>{t("Civic Innovation Corps @ NYC MOCTO")}</Title>
-        <PrimaryNav items={navItems} />
-      </Header>
+      <ProjectHeader />
```

Comment:
I like how you moved the header and footer into their own components! It really helps with readability and makes things more modular